src/WebJobs.Script/FileTraceWriter.cs

```diff
@@ -38,9 +40,17 @@ public FileTraceWriter(string logFilePath, TraceLevel level) : base(level)
             }
             else
             {
```

Comment:
Ah, I completely misread the type and thought it was an override :+1: 
Looks good!